src/components/Form.js

```diff
@@ -249,7 +250,7 @@ class Form extends React.Component {
     render() {
         return (
             <>
-                <ScrollView
+                <ScrollViewWithContext
```

Comment:
This change caused this issue: https://github.com/Expensify/App/issues/13909

Seems to be related to the ref